### PR TITLE
fix: replace node_pools/node_pool_size with worker_pools/size

### PR DESCRIPTION
**Pull request by rodrigc**
Signed-off-by: Craig Rodrigues <craig@quiknode.io>

### CODE REVIEW

**Comment by devoncrouse** [comment]
Thanks for the fix! Also fyi, there's a bit of workers submodule documentation started [here](https://oracle-terraform-modules.github.io/terraform-oci-oke/guide/workers.html) for the 5.x pre-release that may be of use, pending addition of imported [examples](https://github.com/oracle-terraform-modules/terraform-oci-oke/tree/5.x/examples) and other updates.

**Comment by rodrigc** [comment]
Thanks, I'll help out where I can.
I'm using this terraform module and it is excellent!

File: docs/src/guide/topology.md [hunk 1]
```diff
@@ -164,22 +164,22 @@ Node pools enable you to create pools of machines within a cluster that have dif
...
 When using this module to create the node pools, the following is done:
 
-* a number of node pools are created. The number of node pools created is equal to the number of elements in the node_pools parameter e.g.
+* a number of node pools are created. The number of node pools created is equal to the number of elements in the worker_pools parameter e.g.
 
 ```
 worker_pools = {
-  np1 = { shape = "VM.Standard.E4.Flex", ocpus = 1, memory = 16, node_pool_size = 1, boot_volume_size = 150, autoscale = true, label = { app = "frontend", pool = "np1" } }
-  np2 = { shape = "VM.Standard.E4.Flex", ocpus = 1, memory = 16, node_pool_size = 1, boot_volume_size = 150, autoscale = true, label = { app = "frontend", pool = "np1" } }
+  np1 = { shape = "VM.Standard.E4.Flex", ocpus = 1, memory = 16, worker_pool_size = 1, boot_volume_size = 150, autoscale = true, label = { app = "frontend", pool = "np1" } }
```
Comment (devoncrouse) [inline docs/src/guide/topology.md:171]: This can be `size` at the pool level, omitting the `worker_pool` prefix.

Comment (rodrigc) [reply docs/src/guide/topology.md]: Fixed

**Review by devoncrouse — APPROVED** [review]

**Comment by rodrigc** [comment]
Any idea why this PR still displays:

```
Merging is blocked
The base branch requires all commits to be signed.
```

I thought all my commits were signed?

**Comment by rodrigc** [comment]
@devoncrouse By the way, I tried to use the 5.x branch of this module, but got an API error when creating the node pool.  Any idea as to what I am doing wrong?
See: https://github.com/oracle-terraform-modules/terraform-oci-oke/issues/745

**Comment by hyder** [comment]
Hey @Djelibeybi, 

Can you please look into this?

**Comment by rodrigc** [comment]
Not needed any more

**Pull request closed without merging**